.gitignore

```diff
@@ -1,2 +1,3 @@
 kubectl-gs*
 bin-dist
+.DS_Store
```

Comment:
You can add this to your global gitignore https://gist.github.com/subfuzion/db7f57fff2fb6998a16c